### ISSUE TITLE
Documentation duplicate typo - possibly? Please confirm

### ISSUE THREAD

**Issue by lesleyannjordan** [IBM/build-blockchain-insurance-app#82]
Wrong assignation of the Root Certificate Authority
Description: I am following the tutorial below,

https://github.com/IBM/build-blockchain-insurance-app/blob/master/README.md#step-7-Deploy-Insurance-Smart-Contract-on-the-network

where in the,

Step 3. Build a network - Create MSP Definitions

I found that the text,

Create the "repair shop MSP" definition (same process as shown in gif above)

 

-    Navigate to the Organizations tab in the left navigation and click Create MSP definition.
-     Enter the MSP Display name as Repair Shop MSP and an MSP ID of repairshopmsp.
-     Under Root Certificate Authority details, specify the peer CA that we created _**Shop CA**_ as the root CA for the organization.
-     Give the Enroll ID and Enroll secret for your organization admin, repairShopAdmin and repairShopAdminpw. Then, give the Identity name, Repair Shop Admin.
-     Click the Generate button to enroll this identity as the admin of your organization and export the identity to the wallet. Click Export to export the admin certificates to your file system. Finally click Create MSP definition

where I think it must be,

    Under Root Certificate Authority details, specify the peer CA that we created "**_Repair Shop CA_**" as the root CA for the organization.

In other words, replace "Shop CA" by "Repair Shop CA". 
The same is for Create the police MSP definition, i.e. replace "Shop CA" by "Police CA". 

Please confirm if the Readme text is wrong and update if appropriate.

**Comment by horeaporutiu** [comment]
Yes, that is correct. I will update the README now. Thank you for bringing this up.

**Comment by horeaporutiu** [comment]
Updated.
